packages/chakra-ui-core/src/components/theme-editor/ThemeEditor.tsx

```diff
@@ -11,10 +11,17 @@ type ThemeEditorChild = React.ReactElement<
 
 export type ThemeEditorProps = {
   children: ThemeEditorChild[] | ThemeEditorChild
+  isThemeEditorDrawerOpen?: boolean
 }
 
-export const ThemeEditor: FC<ThemeEditorProps> = ({ children }) => {
-  const { isOpen, onOpen, onClose } = useDisclosure()
+export const ThemeEditor: FC<ThemeEditorProps> = ({ children, isThemeEditorDrawerOpen }) => {
+  const [isOpen, setIsOpen] = useState(isThemeEditorDrawerOpen)
+  const onOpen = () => setIsOpen(true)
+  const onClose = () => setIsOpen(false)
```

Comment:
Please avoid non-memoized functions, we accept code only if it is properly memoized.

Instead of: `const onOpen = () => setIsOpen(true)` you can write: `const onOpen = useCallback(() => setIsOpen(true), [])` and it will be accepted.

Many thanks!